hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOmSnapshot.java

```diff
@@ -412,6 +413,20 @@ public void testListAddNewKey()
 
   }
 
+  @Test
+  public void testNonExistentBucket()
+          throws Exception {
+    String volume = "vol-" + RandomStringUtils.randomNumeric(5);
+    String bucket = "buc-" + RandomStringUtils.randomNumeric(5);
+    //create volume but not bucket
+    store.createVolume(volume);
+    OzoneVolume vol = store.getVolume(volume);
```

Comment:
`M D DLS: Dead store to vol in org.apache.hadoop.ozone.om.TestOmSnapshot.testNonExistentBucket()  At TestOmSnapshot.java:[line 423]`  , This line can be removed

Reply:
Thanks for pointing it out. Removed the line.